student_software_developers.md

```diff
@@ -0,0 +1,21 @@
+# Student Software Developers
+
+## Program
+
+Our goal in this program is to fully integrate undergraduate developers into our team. These developers will be paired with members of DLS to initially build skills and then work issues to advance our software.
+
+## Student Employee Policy
+There is a library-wide [Student Employee Policy](https://pulstaff.princeton.edu/working-pul/library-student-employee-policy/) on the Connecting staff Intranet site.
+
+## Timesheets
+
+Student Software Developers are paid biweekly. Timesheet and pay schedules are available on the [Payroll Cycles website](https://finance.princeton.edu/payroll-labor-accounting/payroll/payroll-basics/monthly-and-biweekly-payroll-cycles)
+
+## Schedules
+
+Student Software Developer schedules are tracked in a shared calendar called "DRDS
```

Comment:
I couldn't find this shared calendar in Outlook. Are the sharing permissions set properly?

Reply:
@sdellis I updated this since there's a new calendar now. @tpendragon is there anything you want to add about how the calendar is administered?

Reply:
I added exchange account info.